### PR TITLE
Updated evn variable format

### PR DESCRIPTION
**Pull request by irfanmurtaza-el**
Frontend wasn't picking Variables from ENV, So updated format to let frontend pick from ENV

### CODE REVIEW

File: app/credentials/page.tsx [hunk 1]
```diff
@@ -105,10 +105,10 @@ export default function CredentialsPage() {
           </h2>
           <Dialog>
             <DialogTrigger asChild>
-              <Button variant="outline">
+              {/* <Button variant="outline">
```
Comment (naqi) [inline app/credentials/page.tsx:108]: Please remove these

File: app/page.tsx [hunk 2]
```diff
@@ -240,7 +240,7 @@ const Page = () => {
         } catch (error) {
           setLoading(false);
           setError('An error occurred while fetching the data. Please try again.');
-          console.log('error', error);
+          console.log('\n\nCHAT_API_ERROR_FRONTEND\n\n', error, `\n\n`);
```
Comment (naqi) [inline app/page.tsx:243]: Do we need new space in logs?

File: prisma/schema.prisma [hunk 3]
```diff
@@ -7,8 +7,8 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = env("DATABASE_URL")
-  // directUrl         = env("DIRECT_URL")
+  url      = env("NEXT_PUBLIC_DATABASE_URL")
+  // directUrl         = env("NEXT_PUBLIC_DIRECT_URL")
```
Comment (naqi) [inline prisma/schema.prisma:11]: can you remove this comment?

File: components/ui/dialog.tsx [hunk 4]
```diff
@@ -11,11 +11,11 @@ const Dialog = DialogPrimitive.Root
 const DialogTrigger = DialogPrimitive.Trigger
 
 const DialogPortal = ({
-  className,
+  // className,
   children,
   ...props
 }: DialogPrimitive.DialogPortalProps) => (
-  <DialogPrimitive.Portal className={cn(className)} {...props}>
```
Comment (naqi) [inline components/ui/dialog.tsx:18]: why did you remove this?

File: app/api/chat/route.ts [hunk 5]
```diff
@@ -97,7 +98,7 @@ export async function POST(request: NextRequest) {
       },
     })
   } catch (error: any) {
-    console.log("error", error)
+    console.log('\n\nCHAT_API_ERROR_BACKEND\n\n', error, `\n\n`);
```
Comment (naqi) [inline app/api/chat/route.ts:101]: Do we need new line in logs?

File: app/page.tsx (hunk 2 above)
Comment (irfanmurtaza-el) [reply app/page.tsx:243]: This will enhance the readability of logs

File: app/api/chat/route.ts (hunk 5 above)
Comment (irfanmurtaza-el) [reply app/api/chat/route.ts:101]: Yes, This will enhance the readability of logs

File: components/ui/dialog.tsx (hunk 4 above)
Comment (irfanmurtaza-el) [reply components/ui/dialog.tsx]: As it was giving Error on local system, now I kept the older version of pnpm-lock file, it's not giving error anymore.

File: prisma/schema.prisma (hunk 3 above)
Comment (irfanmurtaza-el) [reply prisma/schema.prisma:11]: Done

File: app/credentials/page.tsx (hunk 1 above)
Comment (irfanmurtaza-el) [reply app/credentials/page.tsx:108]: Done

**Review by naqi — COMMENTED** [review]
looks good
